kafka/kafka.sh

```diff
@@ -25,6 +25,7 @@ readonly ROLE="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
 readonly RUN_ON_MASTER="$(/usr/share/google/get_metadata_value attributes/run-on-master || echo false)"
 readonly KAFKA_ENABLE_JMX="$(/usr/share/google/get_metadata_value attributes/kafka-enable-jmx || echo false)"
 readonly KAFKA_JMX_PORT="$(/usr/share/google/get_metadata_value attributes/kafka-jmx-port || echo 9999)"
+readonly DATAPROC_IMAGE="$(/usr/share/google/get_metadata_value attributes/image-version)"
```

Comment:
Why is this added? It's not consistent with how other init actions get the image version.

Reply:
Okay got it. We have the DATAPROC_IMAGE_VERSION for this. No need to define it under the metadata.